examples/MINLPs/mult4.jl

```diff
@@ -0,0 +1,2804 @@
+#= 
+    1. Using the below raw expression interface for dense nonlinear problems significantly 
+      reduces JuMP's building time
+    2. `linking_constraints = true` is necessary to be able to solve this dense quadrilinear instance
+=#
+
+replace_expr(expr::Any, x) = expr
+function replace_expr(expr::Expr, x)
+    if Base.Meta.isexpr(expr, :ref)
+        return x[expr.args[2]]
+    end
+    for i in 1:length(expr.args)
+        expr.args[i] = replace_expr(expr.args[i], x)
+    end
+    return expr
+end
+
+function m_10_4_10_100_1(; solver = nothing)
```

Comment:
@odow  This instance now loads instantaneously in JuMP

Reply:
What manual work was involved? I looked into this somewhere right? I can't find the link.

Reply:
Yes! You made this fix in an issue somewhere. Manual work is about explicitly writing every constraint into a raw NL expression. I guess there isn't much of a workaround for it.